cloud/amazon/ec2_vol.py

```diff
@@ -395,8 +500,11 @@ def main():
     name = module.params.get('name')
     instance = module.params.get('instance')
     volume_size = module.params.get('volume_size')
+    volume_type = module.params.get('volume_type')
```

Comment:
We don't need to obtain `volume_type`, `iops` or `delete_on_termination` in `main()` - they don't seem to be used.